js/graphdata.tsv

```diff
@@ -0,0 +1,356 @@
+datestamp	certsIssued	certsActive	fqdnsActive	regDomainsActive
+2015-09-12	1	NULL	NULL	NULL
```

Comment:
There's no problem, it's just expensive to calculate. The NULL values are ones where I have not rewound time to derive the data. I picked a series of dates ~45 days apart to provide historical context, and going forward the data will be day-to-day.